src/app/static/src/app/store/root/getters.ts

```diff
@@ -119,3 +133,11 @@ const allReviewInputsWarnings = (state: RootState) => {
     return [...sapWarnings, ...genericChartWarnings]
 }
 
+const getVersionNotes = (versions: Version[], name: string) => {
+    return versions.reduce((result: Note[], versionNote) => {
+        const note = versionNote.note || ""
+        const updated = versions && formatToISODateTime(versionNote.updated)
+
+        return result.concat({note, updated, name})
```

Comment:
I think "name" here should probably be the version marker, or project name plus version marker e.g. My Project v2. Otherwise all the note sections just get labelled with the project name.